data-science/us-east-1/genai-llm-rag-bedrock-poc/locals.tf

```diff
@@ -0,0 +1,57 @@
+locals {
+
+  name = "${var.project}-${var.environment}-genai-llm-rag-bedrock-poc"
+  tags = {
+    Terraform   = "true"
+    Environment = var.environment
+  }
+
+  container_definitions = {
+    b2chat = {
+      image                     = "905418344519.dkr.ecr.us-east-1.amazonaws.com/bb-data-science-genai-llm-rag-bedrock-poc-b2chat:latest"
+      enable_cloudwatch_logging = false
+      readonlyRootFilesystem    = false
+      cpu = 1024
+      memory = 2048
```

Comment:
@martingaleano does this container require this much resources?

Reply:
@diego-ojeda-binbash  No, it doesn't. I've just updated the PR and replaced those values with cpu = 512 and memory = 1024. I think  these values fit much better.